pyramid_zipkin/tween.py

```diff
@@ -106,7 +113,7 @@ def zipkin_tween(handler, registry):
     def tween(request):
         zipkin_settings = _get_settings_from_request(request)
 
-        with zipkin_span(
+        tween_kwargs = dict(
```

Comment:
use dict comprehension since we target python >= 2.7

Reply:
The reason I didn't use a dict comprehension is because I didn't want to wrap every key in quotes, since they need to be strings.